src/site/content/en/blog/streams/index.md

```diff
@@ -273,38 +273,38 @@ console.log(`The stream is ${locked ? 'indeed' : 'not'} locked.`);
 ### Readable stream code samples
 
 The code sample below shows all the steps in action. You first create a `ReadableStream` that in its
-`underlyingSource` argument defines a `start` method that tells the stream's `controller` to
+`underlyingSource` argument (that is, the `TimestampSource` class) defines a `start` method
+that tells the stream's `controller` to
 `enqueue()` a timestamp every second during ten seconds and then to `close()` the stream. You consume this
 stream by creating a reader via the `getReader()` method and calling `read()` until the stream is
 `done`.
 
 ```js
-const startEnqueuing = (controller) => {
-  let interval = null;
-  interval = setInterval(() => {
-    const string = new Date().toLocaleTimeString();
-    // Add the string to the stream.
-    controller.enqueue(string);
-    console.log(`Enqueued ${string}`);
-  }, 1_000);
-
-  setTimeout(() => {
-    clearInterval(interval);
-    // Close the stream after 10s.
-    controller.close();
-  }, 10_000);
-};
+class TimestampSource {
+  #interval
```

Comment:
Please stick to plain JavaScript.

Reply:
This is the new [private class fields](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields) notation.